docs/naivebayes.md

```diff
@@ -0,0 +1,138 @@
+Naive Bayes Classifiers
+=======================
+
+The Naive Bayes classifier is a simple probabilistic classification model based on Bayes Theorem. It is a simple and fast classification model. Since Naive Bayes classifiers simply classifies sets of data by which class has the highest conditional probability, Naive Bayes classifiers can use any distribution or model which has a probabilistic interpretation of the data as one of its components. Basically if it can output a log probability, then it can be used in Naive Bayes.
```

Comment:
"It is a simple and fast classification model" is redundant with the previous sentence, just remove. Using 'simple' too much, "Since Naive Bayes classify data using conditional probability,..."